### ISSUE TITLE
Images without alt text should have alt set to null

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#4557]
When an image does not have alt text, it should be given an attribute of `alt="null"`. A screen reader will bypass the image in the tab order and avoid an additional step with no information.

I just learned this!

In the default theme, where alt text doesn't exist we've been removing the `alt` attribute. We should change that.

**Comment by mtub** [comment]
Thanks for sharing, now I got a chance to learn that, too.

But it still would be a good thing to have alt texts as often as possible, right? Or do you suggest to intentionally leave alt texts empty and skip the images in screen readers with the help of this attribute?

**Comment by NateWr** [comment]
**Always** better to have `alt` text for images except when:

- the image is purely decorative and has no "content" (example - [patterns](https://www.toptal.com/designers/subtlepatterns/).
- the image is sufficiently described in other text.

**Comment by NateWr** [comment]
Places to look for alt tags:

- Journal logo
- Journal homepage image
- Issue cover image
- Article cover image

**Comment by NateWr** [comment]
@thinkbulecount2 (see previous message)

**Comment by thinkbulecount2** [comment]
Pull requests here, @NateWr:
https://github.com/pkp/ojs/pull/2361
https://github.com/pkp/pkp-lib/pull/4681

**Comment by thinkbulecount2** [comment]
ojs PR: https://github.com/pkp/ojs/pull/2361
omp PR: https://github.com/pkp/omp/pull/668

**Comment by thinkbulecount2** [comment]
@NateWr made the changes you requested!

**Comment by NateWr** [comment]
Great! Let me know when the tests pass and I'll merge them! :+1:

**Comment by NateWr** [comment]
All merged. :tada:

**Comment by NateWr** [comment]
This information was incorrect. When alt text is missing the attribute should be included with no text. Example: `alt=""`. This was fixed in the commits above.
